networkx/algorithms/bipartite/matrix.py

```diff
@@ -133,7 +133,7 @@ def from_biadjacency_matrix(A, create_using=None, edge_attribute='weight'):
     0 or 1 representing membership in part 0 or part 1 of the bipartite graph.
 
     If `create_using` is an instance of :class:`networkx.MultiGraph` or
-    :class:`networkx.MultiDiGraph` and the entries of `A` are of type ``int``,
```

Comment:
I looked hard at this and am willing to change it. The examples are not clear. The spec is not clear. In the end I decided that it doesn't confuse anything to leave it as unformatted and makes it slightly easier to read in the raw state.

Notice that not all python code gets formatted with double ticks. Only expressions that are long enough (?meaning personal choice?).